financialaid/migrations/0003_added_timestamps.py

```diff
@@ -4,7 +4,7 @@
 
 import datetime
 from django.db import migrations, models
-
+import pytz
```

Comment:
I don't think so, all `DateTimeField`s will have a timezone set since `USE_TZ = True` in the settings, even if they were set with naive `datetime` object. For example:

    In [8]: n = datetime.now()
    
    In [9]: n
    Out[9]: datetime.datetime(2017, 5, 25, 14, 17, 24, 487031)
    
    In [10]: n.tzinfo
    
    In [11]: p.updated_on = n
    
    In [12]: p.save()
    
    In [13]: p.refresh_from_db()
    
    In [14]: p.updated_on
    Out[14]: datetime.datetime(2017, 5, 25, 14, 17, 34, 931526, tzinfo=<UTC>)